spec/haml_lint/linter/spaces_for_indentation_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe HamlLint::Linter::SpacesForIndentation do
+  include_context 'linter'
+
+  # rubocop and overcomit HardTab checks don't like tabs either, but they don't
+  # check for hex, so use that to avoid triggering them
+  let(:tab) { "\x09" }
+
+  context 'when line contains just a tab in its indentation' do
+    let(:haml) { <<-HAML }
+      %span
+#{tab * 4}- bar
+    HAML
+
+    it { should report_lint line: 2 }
+  end
+
+  context 'when line contains spaces and a tab in its indentation' do
```

Comment:
This test isn't necessary because the HAML parser will report an error if tabs and spaces are mixed, e.g.

```
Inconsistent indentation: "  \t" used for indentation, but the rest of the document was indented using 2 spaces.
```